pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[tool.poetry]
+name = "wkcuber"
+version = "0.0.0"
```

Comment:
The correct version is read from the GH releases page through `setuptools_scm`, right? Can we omit this property completely here? Otherwise, maybe add comment mentioning `setuptools_scm`.